src/runtime.js

```diff
@@ -87,6 +88,7 @@ export function installRuntimeServices(global) {
   installViewerService(global);
   installViewportService(global);
   installHistoryService(global);
+  installStorageService(global);
```

Comment:
@cramforce should we push promisable services to be installed in a micro-task here so that we don't block the initial render?